test/conformance/kuttl/cleanup/clusterpolicy/context-cleanup-pod/03-policy.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+apply:
+- policy.yaml
+assert:
+- policy.yaml
```

Comment:
Do we have ready status for the cleanup policy?

Reply:
No, We don't have `status` as ready in cleanup policy.